### PR TITLE
variables and functions

### PR DESCRIPTION
**Pull request by ChrisNewmanGDL**
Hi, the code ran ok, but I didn't understand the warnings I got -  "implicit conversion loses floating point precision"
I get that double has more precision than float, but dont know when is it best to use double or float?

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -80,43 +95,79 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+}
 /*
  6)
  */
-
+bool validSubmission(int wordCount, float lineSpacing, bool headerCorrect)
+{
+ignoreUnused(wordCount, lineSpacing, headerCorrect);
+return{};
+}
```
Comment (matkatmusic) [inline main.cpp]: Fix this indentation
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -80,43 +95,79 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(averageSpeed, distanceInMiles);
+    return{};
+}
 /*
  9)
  */
-
+void addChordTones(float inputNoteFrequency, float third = 0.333, float fifth = 0.583)
+{
+    ignoreUnused(inputNoteFrequency, third, fifth);
```
Comment (matkatmusic) [inline main.cpp]: you are initializing your `float` function arguments with doubles.
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -137,27 +188,27 @@ int main()
...
+    grandaughterTantrumLikelihood (3, 1);
     //4)
-    
+    bool playNote = triggerActivated(7);
     //5)
-    
+    bool blowWhistle = matchCanStart(11);
     //6)
-    
+    bool markThis = validSubmission(2000,1.5, 1);
```
Comment (matkatmusic) [inline main.cpp]: Be consistent with your spacing around your function parameters.
- [ ] revise

File: main.cpp [hunk 4]
```diff
@@ -137,27 +188,27 @@ int main()
...
+    float podcastLength = journeyTime(60, 225);
     //9)
-    
+    addChordTones(1.167);
     //10)
+    meetingQuorate(7,13);
     
-    
-    ignoreUnused(carRented);
+    ignoreUnused(carRented,pitchArea,phraseLength,grandaughterTantrumLikelihood, playNote, blowWhistle, markThis, enterPlayground, podcastLength, addChordTones, meetingQuorate);
```
Comment (matkatmusic) [inline main.cpp]: Be consistent with your spacing around your function parameters
Consult the coding style example in the Readme.MD if you aren't sure what is wrong with these lines of code with regard to the coding style.
- [ ] revise

File: main.cpp [hunk 5]
```diff
@@ -137,27 +188,27 @@ int main()
...
     
     //1)
-    
+    auto pitchArea = rectangleArea(30, 121.5);
     //2)
-    
+    auto phraseLength =segmentLength(22, 45);
     //3)
-    
+    grandaughterTantrumLikelihood (3, 1);
```
Comment (matkatmusic) [inline main.cpp]: Be consistent with your spacing around your function parameters
Consult the coding style example in the Readme.MD if you aren't sure what is wrong with these lines of code with regard to the coding style.
- [ ] revise

File: main.cpp [hunk 6]
```diff
@@ -60,10 +60,25 @@ void variableDeclarations()
...
+char axis = 'y';
+bool OnOff = false; 
+bool answer = true;
+bool upDown = true;
+float height = 1.80f;
+float weight = 6.54f;
+float width = 4.356f;
+double kmDriven = 8.3333;
+double hardSumsResult = 34.502935409324809834;
+double minisculeThingSize = 0.0000000000234;
```
Comment (matkatmusic) [inline main.cpp]: Fix this indentation
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
You have lots of code formatting issues.
Always strive to make your code as easy to read as possible by making sure it is consistently formatted and indented.
I have marked specific spots that need attention.
- [ ] revise

File: main.cpp (hunk 6 above)
Comment (matkatmusic) [reply main.cpp]: All of these variables need to be indented correctly.
Line them up with `int number = 2;`
Please take a look at the Readme.MD.  There is a coding standard example that demonstrates the coding standard for the course. 
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 6 above)
Comment (ChrisNewmanGDL) [reply main.cpp]: ok - I had an eye test the other day too!

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**